js/formValidation.js

```diff
@@ -2,21 +2,14 @@ const formElement = document.getElementById('form');
 const email = document.getElementById('email');
 const small = document.querySelector('small');
 
-const regexCode = /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$/i;
-function isEmailValid(email) {
-  return regexCode.test(email);
-}
-
-function checkInputs() {
-  const emailValue = email.value.trim();
-  if (emailValue !== '' && emailValue === emailValue.toLowerCase()) {
-    isEmailValid(emailValue);
-  } else {
-    small.innerHTML = 'Email Must be all lowercase.';
-  }
-}
+const isLowerCase = (str) => /[a-z]/.test(str) && !/[A-Z]/.test(str);
```

Comment:
- [ ] Good job so far but the email validation is not working as I can currently submit the form with an email having a capital letter. Verify that your lower case function is working properly for a better user interface.

Reply:
Thanks @Kingstalux 
I will work on it and comment down here please before I request for review because am having only one review

Reply:
okay @sainingo I'll be waiting

Reply:
Hello @Kingstalux kindly have a look at it have tried to implement the changes
Thank you

Reply:
okay let check it now

Reply:
Okay, @sainingo I've cloned your project and tested it and it works okay now. You can submit.
![Annotation 2022-03-10 183328](https://user-images.githubusercontent.com/85113567/157721619-a3bec69d-3342-42b2-9e46-d209e5d13687.jpg)